run_acceptance_tests.sh

```diff
@@ -1,11 +1,30 @@
 #!/bin/bash -e
 ENV=${1:-local}
-BROWSER=${2:-chrome}
+BROWSER=${2:-remote-chrome}
+
+# sm ports needed when running assets-frontend locally via service manager
+sm_port_mapping=$(sm -s | grep PASS | awk '{ print $12"->"$12 }' | paste -sd "," -)
+
+# port mapping for running contact-frontend service
+service_port_mapping="6001->6001,"
+port_mappings="$service_port_mapping$sm_port_mapping"
+
+IMAGE=artefacts.tax.service.gov.uk/chrome-with-rinetd:latest
```

Comment:
Just a heads up that by hard coding the browser image to chrome-with-rinetd the `BROWSER` argument becomes redundant, it could lead to confusion if someone runs the script `./run_acceptance_tests.sh local remote-edge` for example. I'd personally either keep the [original IF statement](https://github.com/hmrc/platform-example-ui-journey-tests/blob/974bf808f9e95aef9c803f7e837b9c91bd814c15/run_browser_with_docker.sh#L29) or hard code `BROWSER=remote-chrome` on L3.

Reply:
@tmc-ee Changed as per your suggestion.